TabloidCLI/Repositories/PostRepository.cs

```diff
@@ -107,7 +107,28 @@ public List<Post> GetByAuthor(int authorId)
 
         public void Insert(Post post)
         {
-            throw new NotImplementedException();
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = $@"INSERT INTO Post(Title, URL, PublishDateTime, AuthorId, BlogId)
+                                        OUTPUT INSERTED.Id
+                                        VALUES(@title, @url, @publish, @authorId, @blogId)";
```

Comment:
blogId is not being used?

Reply:
Because I'm dumb and it's fixed in the new push.